windows/utils/win_install_online_updates.yml

```diff
@@ -11,6 +11,8 @@
 #     Value '*' matches all categories.
 #   win_updates_reject_list (optional): A list of update titles or KB numbers that
 #     can be used to specify which updates are to be excluded from installation.
+#   win_updates_reboot(optional): If set to true, Ansible will automatically reboot the remote host
```

Comment:
The format need to be consistent with others, e.g., space before (optional), period at the end.

Reply:
Updated. Thanks.